marconi-mamba/src/Marconi/Api/Types.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeApplications      #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE TypeOperators         #-}
+{-# LANGUAGE UndecidableInstances  #-}
+
+-- |
+-- This module provides support for writing handlers for JSON-RPC endpoints
+module Marconi.Api.Types where
+
+import Control.Lens (makeClassy)
+import Marconi.IndexersHotStore (IndexerHotStore)
+import Network.Wai.Handler.Warp (Settings)
+
+
+-- | Typre represents mamba specific marconi indexer cache
```

Comment:
```suggestion
-- | Type represents mamba specific marconi indexer cache
```

Reply:
I think this comment needs to be removed?

Reply:
yes, thnx